helix-core/src/lib.rs

```diff
@@ -34,6 +34,20 @@ pub fn find_first_non_whitespace_char(line: RopeSlice) -> Option<usize> {
     line.chars().position(|ch| !ch.is_whitespace())
 }
 
+pub fn find_last_non_word_char(line: RopeSlice) -> Option<usize> {
+    line.to_string().chars()
```

Comment:
Maybe it would make more sense to first reverse the iterator: https://docs.rs/ropey/1.3.1/ropey/iter/struct.Chars.html#impl

Reply:
Reverse  seem is not fit the case